evm/deploy-gateway-v5.0.x.js

```diff
@@ -2,6 +2,18 @@
 
```

Comment:
here's how to test upgrade in online mode

1. Set test key in .env along with ADMIN_ADDRESSES=["test account"], and ADMIN_THRESHOLD=1
2. Deploy an v4.3 gateway, `node evm/deploy-gateway-v4.3.x.js -e testnet -n avalanche`
3. Deploy a new gateway while reusing proxy, `node evm/deploy-gateway-v5.0.x.js -e testnet -n avalanche -g test_account -m test_account --reuseProxy`
4. Upgrade gateway to the new implementation, `node evm/deploy-gateway-v5.0.x.js -e testnet -n avalanche -u`

After the changes, this should work as well as the offline mode